dts/bindings/gpio/microchip,mec5-gpio.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2024, Microchip Technology Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Microchip MEC5 GPIO node
+
+compatible: "microchip,mec5-gpio"
+
+include: [gpio-controller.yaml, base.yaml]
+
+properties:
+  reg:
+    required: true
+
+  port-id:
+    type: int
+    required: true
+    description: Zero based GPIO port number
```

Comment:
Port numbers seems to be a HAL'ism, not a property of the hardware IP itself. Please see if you can work-around having this set in the devicetree nodes.

Reply:
Removed port-id and girq-id (unused) properties.